### PR TITLE
Fix derived channel units issue

### PR DESCRIPTION
**Pull request by evanmayer**
We observed in a test that the units of some current/voltage monitoring channels were not as expected. The issue was that the wrong variable was passed into a macro that converts the units to the final format for display in the telemetry viewer, `kst`. This change fixes issue #42.

### CODE REVIEW

File: blast_config/derived.c [hunk 1]
```diff
@@ -594,5 +592,4 @@ derived_tng_t derived_list[] = {
     BITWORD("TRIGGER_XSC1", "trigger_xsc", 1, 1),
     BITWORD("TRIGGER_STATE_XSC", "trigger_xsc", 2, 6),
 
-  END_OF_DERIVED_CHANNELS
-};
+    END_OF_DERIVED_CHANNELS};
```
Comment (ianlowe13) [inline blast_config/derived.c:595]: As long as this is allowed by our style checker everything looks good to me

Comment (evanmayer) [reply blast_config/derived.c:595]: The target containing this file was checked during a passing CI build: https://github.com/tim-balloon/TIMflight/actions/runs/4801605669/jobs/8544038607#step:5:1665

Comment (ianlowe13) [reply blast_config/derived.c:595]: Ah, right I forgot that this gets compiled in the checks.

**Review by ianlowe13 — APPROVED** [review]
Checks out

**Pull request merged**